src/HashGenerator.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\FocusContactCenterBundle;
+
+class HashGenerator
+{
+    /**
+     * @var Configuration
+     */
+    private $configuration;
+
+    public function __construct(Configuration $configuration)
+    {
+        $this->configuration = $configuration;
+    }
+
+    public function hash(string $change): string
+    {
+        [$login, ] = explode('@', $this->configuration->getLogin());
```

Comment:
After you add a DependecyInjection for configuration, please check that you have a `@` at login.